src/web/lib/components/Banner/index.scss

```diff
@@ -12,8 +11,7 @@ $banner-height: 71px;
   height: $banner-height;
   position: fixed;
   width: 100%;
-  box-shadow: 0 0 0 1px rgba(0, 0, 0, .05),
-              0 0 10px rgba(0, 0, 0, .05);
+  box-shadow: 0 0 0 1px rgba(0, 0, 0, .05), 0 0 10px rgba(0, 0, 0, .05);
```

Comment:
I think Sass supports named colors and hex colors for `rgb()` and `rgba()` so we _might_ be able to tighten this up slightly by going with:

```sass
box-shadow: 0 0 0 1px rgba(#000, .05), 0 0 10px rgba(#000, .05);
```